test/integration/test-amp-carousel.js

```diff
@@ -16,11 +16,15 @@
 
 import {createFixtureIframe, expectBodyToBecomeVisible} from
     '../../testing/iframe';
+//import * as sinon from 'sinon';
 
 describe('integration amp-carousel', () => {
 
   let fixture;
+  //let sandbox;
```

Comment:
haha yeah, meant to leave it in intentionally. as my tests are all red locally for carousels. will remove and move it over to a different branch.